docs/jwst/stpipe/config_asdf.rst

```diff
@@ -75,3 +268,28 @@ The one keyword to explain further is ``reftype``. This is what CRDS uses to
 determine which reference file is being sought after. For step configurations,
 this has the format ``pars-<step_name>`` where ``<step_name>`` will be the Python
 class name, in lowercase.
+
+
+History
+~~~~~~~
+
+Parameter reference files also require at least one history entry. This can be found in the ``history`` block under ``entries``:
+
+.. code-block::
+
+    history:
+      entries:
+      - !core/history_entry-1.0.0 {description: Base values, time: !!timestamp '2019-10-29
+          21:20:50'}
+
+It is highly suggested to use the Python API to add history entries::
+
+.. doctest_skip::
+
+   >>> import asdf
+   >>> cfg = asdf.open('config.asdf')
+       #
+       # Modify cfg['parameters'] as necessary
+       #
+   >>> cfg.add_history_entry('Parameters modified for some reason')
+   >>> cfg.write_to('config_modified.asdf')
```

Comment:
This block should be something like:
```
    import asdf
    cfg = asdf.open('config.asdf') # doctest: +SKIP
    #
    # Modify cfg['parameters'] as necessary
    #
    cfg.add_history_entry('Parameters modified for some reason') # doctest: +SKIP
    cfg.write_to('config_modified.asdf') # doctest: +SKIP
```
and no need for the `.. doctest_skip::` directive.

Reply:
The `# doctest: +SKIP` won't show up in the final HTML document.